### PR TITLE
Fix tests for testing encrypt options

### PR DESCRIPTION
**Pull request by lilgreenbird**

### CODE REVIEW

File: src/test/java/com/microsoft/sqlserver/jdbc/fips/FipsTest.java [hunk 1]
```diff
@@ -86,7 +86,7 @@ public void fipsPropertyTest() throws Exception {
         props.remove(Constants.ENCRYPT);
         props.remove(Constants.TRUST_SERVER_CERTIFICATE);
 
-        try (Connection con = PrepUtil.getConnection(connectionString, props)) {
+        try (Connection con = PrepUtil.getConnection(connectionString + "encrypt=false", props)) {
```
Comment (Jeffery-Wasty) [inline src/test/java/com/microsoft/sqlserver/jdbc/fips/FipsTest.java:89]:
```suggestion
        try (Connection con = PrepUtil.getConnection(connectionString + ";encrypt=false", props)) {
```

Comment (lilgreenbird) [reply src/test/java/com/microsoft/sqlserver/jdbc/fips/FipsTest.java:89]: no, not trying to change connectionString, just AEConnectionString

Comment (Jeffery-Wasty) [reply src/test/java/com/microsoft/sqlserver/jdbc/fips/FipsTest.java:89]: Not needed.

**Review by Jeffery-Wasty — APPROVED** [review]

File: src/test/java/com/microsoft/sqlserver/jdbc/fips/FipsTest.java (hunk 1 above)
Comment (lilgreenbird) [reply src/test/java/com/microsoft/sqlserver/jdbc/fips/FipsTest.java:89]: hmm this shouldn't be here it's being ignored, should be removed

**Review by Jeffery-Wasty — APPROVED** [review]

**Comment by lilgreenbird** [comment]
@microsoft-github-policy-service agree

**Pull request merged**